blocks/algolia-assortment-content-source-block/README.md

```diff
@@ -0,0 +1,26 @@
+# @wpmedia/algolia-assortment-content-source-block
+
+The Algolia assortment content source block allows you to display products from an Algolia index.
+
+## ANS Schema
```

Comment:
Minor - but algolia is not using ANS Schema

Reply:
ah yeah. one sec

Reply:
updated @matthewroach